pep-0622.rst

```diff
@@ -1390,6 +1390,80 @@ similar.  This example is from ``difflib.py`` in the stdlib::
 
   for tag, _, _, j1, j2 in group: ...
 
+Use some other syntax instead of ``|`` for OR patterns
+------------------------------------------------------
+
+A few alternatives to using ``|`` to separate the alternatives in OR
+patterns have been proposed.  Instead of::
+
+  case 401|403|404:
+      print("Some HTTP error")
+
+the following proposals have been fielded:
+
+- Use a comma::
+
+    case 401, 403, 404:
+      print("Some HTTP error")
+
+  This looks too much like a tuple -- we would have to find a
+  different way to spell tuples, and the construct would have to be
+  parenthesized inside the argument list of a class pattern.  In
+  general, commas already have many different meanings in Python, we
+  shouldn't add more.
+
+- Allow stacked cases::
+
+    case 401:
+    case 403:
+    case 404:
+      print("Some HTTP error")
+
+  This is how this would be done in C, using its fall-through
+  semantics for cases.  However, we don't want to mislead people into
+  thinking that ``match``/``case`` uses fall-through semantics (which
+  are a common source of bugs in C).  Also, this would be a novel
+  indentation pattern, which might make it harder to support in IDEs
+  and such (it would break the simple rule "add an indentation level
+  after a line ending in a colon").
```

Comment:
Also, this alternative wouldn't support nested OR patterns.

Reply:
Thanks, fixed.